about.md

```diff
@@ -4,12 +4,15 @@ title: About
 permalink: /about/
 ---
 
-This is the base Jekyll theme. You can find out more info about customizing your Jekyll theme, as well as basic Jekyll usage documentation at [jekyllrb.com](http://jekyllrb.com/)
+This is the base Jekyll theme. You can find out more info about customizing your Jekyll theme, as well as basic Jekyll usage documentation at [jekyllrb.com](https://jekyllrb.com/)
 
-You can find the source code for the Jekyll new theme at:
-{% include icon-github.html username="jekyll" %} /
+You can find the source code for Minima at GitHub:
+[jekyll][jekyll-organization] /
```

Comment:
That approach was changed upstream to allow users switch themes via GH-Pages.

But this about.md is specific to Minima.
It also serves as a showcase page for what Minima looks like  when used with `jekyll new --`

Confusion.. confusion..